api/IapiFunctions.ts

```diff
@@ -0,0 +1,13 @@
+export interface IAccountData {
+  email: string;
+  password: string;
+}
+export interface IUserWeeklyPick {
+  userId: string;
+  weekNumber: string;
+}
+export interface IWeeklyPicks {
+  gameId: string;
+  gameWeekId: string;
+  userResults: string;
+}
```

Comment:
Just to understand, is this file named `IapiFunctions.ts` so developers know this is where interfaces live?

Reply:
Just to clarify, are the prefixes "I" before all these interfaces like `IapiFunctions.ts` and `IWeeklyPicks` are supposed to represent Interface?

Reply:
Correct. It's a naming convention so that you know it's an Interface.